WooFoundation/WooFoundation/Protocols/Analytics/WooAnalyticsStat.swift

```diff
@@ -0,0 +1,1231 @@
+/// This enum contains all of the events we track in the app. Please reference the "Woo Mobile Events Draft i2"
+/// spreadsheet for more details.
+///
+/// One of goals of this `enum` is to be able to list all the event names that we use throughout
+/// the app. We can also potentially make a parser to gather all the event names and automatically
+/// compare them with WCAndroid. With that, we can make sure both platforms are tracking the
+/// same events. Right now, we use the spreadsheet. XD
```

Comment:
We can update some of these comments:
```suggestion
/// This enum contains all of the events we track in the WooCommerce app.
///
/// One of goals of this `enum` is to be able to list all the event names that we use throughout
/// the app.
```

Reply:
Nice catch! Updated in https://github.com/woocommerce/woocommerce-ios/pull/12685/commits/725cf2a14bc1b3633b2a10caab220e28061dee49